coursedata/texts/en.yaml

```diff
@@ -40,7 +40,11 @@ ui:
     invalid_teacher_invitation_code: "The teacher invitation code is invalid. To become a teacher, reach out to hedy@felienne.com."
     page_not_found: "We could not find that page!"
 ClientErrorMessages:
-    Transpile_success: "Great job!"
+    Transpile_success1: "Good job!"
```

Comment:
Could we make this into a list instead:

Transpile_success = Good job!, Amazing, etc...

In yaml you can make a list like this:
Transpile_success:
      - Good job!
      - Amazing
      - ...

Reply:
That was different languages can also have different lengths of messages, so you'd need to change the code a bit too